include/swift/Remote/MetadataReader.h

```diff
@@ -326,8 +327,10 @@ class MetadataReader {
   }
 
   /// Given a pointer to a known-error existential, attempt to discover the
-  /// pointer to its metadata address and its value address.
-  Optional<std::pair<RemoteAddress, RemoteAddress>>
+  /// pointer to its metadata address, its value address, and whether this
+  /// is a toll-free-bridged NSError or an actual Error existential wrapper
+  /// around a native Swift value.
+  Optional<std::tuple<RemoteAddress, RemoteAddress, bool>>
```

Comment:
Consider just making a `struct`, especially since this type is used multiple places throughout the reader.